kolibri/plugins/style_guide/assets/src/views/content/api_table.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+
+  <div>
+
+    <h2>Name: {{ api.name }}</h2>
+    <h2>Description: {{ api.description }}</h2>
+
+    <div v-if="api.props.length">
+      <h2>Props</h2>
+      <table>
+      <tr>
+        <th>Name</th>
+        <th>Type</th> 
+        <th>Required</th>
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr v-for="prop in api.props">
+        <td>{{ prop.name }}</td>
+        <td>{{ prop.value.type }}</td>
+        <td>{{ prop.required ? prop.required : 'No'}}</td>
```

Comment:
Actually using the 'required' field directly makes the most sense. Thanks for catching!